content/python/concepts/casting/casting.md

```diff
@@ -36,15 +37,15 @@ This will output:
 
 ## Explicit Type Conversion
 
-In explicit type casting (also known as type conversion), the user converts the data type of variable with the help of predefined functions:
+In explicit type casting, the user converts the data type of a variable using a predefined function. This function is typically the constructor of another data type.
```

Comment:
Wanting to confirm, are we speaking about conversion constructors in this second sentence? Would you mind linking me to your source? Thanks in advance! ☺️